src/store/undoRedo/__tests__/getChangedKeys.test.ts

```diff
@@ -63,6 +63,13 @@ describe('getChangedKeys', () => {
       },
     }
 
-    expect(getChangedKeys(a, b)).toEqual(['foo.bar.baz', 'foo.bar.boo', 'foo.bar.uwu'])
+    expect(getChangedKeys(a, b)).toEqual(['foo.bar.baz~', 'foo.bar.boo-', 'foo.bar.uwu+'])
+  })
+
+  it('should handle deleted array items', () => {
+    const a = {foo: {bar: [1, 2, 3]}}
+    const b = {foo: {bar: [1, 3]}}
```

Comment:
Would it be okay to replace numeric elements with strings in this example? E.g. `['x', 'y', 'z']`.
It would be slightly easier to understand `.toEqual(['foo.bar.1~', 'foo.bar.2-'])` - that is about keys, not values.